crates/neon-runtime/src/napi/error.rs

```diff
@@ -9,3 +16,25 @@ pub unsafe extern "C" fn new_type_error(_out: &mut Local, _msg: Local) { unimple
 pub unsafe extern "C" fn new_range_error(_out: &mut Local, _msg: Local) { unimplemented!() }
 
 pub unsafe extern "C" fn throw_error_from_utf8(_msg: *const u8, _len: i32) { unimplemented!() }
+
+pub unsafe extern "C" fn is_throwing(env: Env) -> bool {
```

Comment:
FYI, this doesn't need to be `extern "C"`. We have this in several places. I created an issue to clean it up.

https://github.com/neon-bindings/neon/issues/548